operator/api/v1alpha1/central_types.go

```diff
@@ -157,6 +157,14 @@ type CentralComponentSpec struct {
 	DeploymentSpec `json:",inline"`
 }
 
+// GetExposure provides a way to retrieve the Exposure setting that is safe to use on a nil receiver object.
+func (c *CentralComponentSpec) GetExposure() *Exposure {
```

Comment:
Maybe I'm looking wrong, but did this method end up unused? If so, please remove.
Ditto for `GetRoute` below.